en/developer_guide/block_display.rst

```diff
@@ -0,0 +1,93 @@
+Block display
+=============
+
+Display Independence
+--------------------
+
+In Front Office, some blocks need specific stylesheets and javascript file.
+To perform this, the main node layout loads the requireJS library which allows each block to call require() function.
+To allow stylesheets dynamic load, Open Orchestra offers an AMD (Asynchronous Module Definition) named openOrchestraCSS
+which is meant to be use with requireJS.
+In a case in which the same block is displayed several times, stylesheets and javascript file corresponding will be
+loaded only once.
+
+blockLoader.js file
+-------------------
+
+For each block there is a blockLoader.js structured as the following:
+
+.. code-block:: javascript
+
+    requirejs(
+        ['jquery'],
+        function() {
+            require(
+                [
+                    //Define required elements
+                    'openOrchestraCss', //Module used to load stylesheets
+                    ''/bundles/nameofbundle/block/NameOfMyBlock/js/foo.js'', //Specific js in block
+                    '/bundles/nameofbundle/libs/bar/bar.js'//js in common libraries
+                ],
+                function (openOrchestraCss) {
+                    //Load block stylesheet
+                    openOrchestraCss.load(['/bundles/nameofbundle/block/NameOfMyBlock/css/style.css']);
+
+                    //You can add optional javascript code here
+                    jQuery(document).ready(function ($) {
+
+
+                    });
+                }
+            )
+        }
+    );
+
```

Comment:
You have said all that was to say, but I wonder if the explanation would be in the code block as comment or outside.